packages/frontend-shared/cypress/support/e2e.ts

```diff
@@ -559,9 +559,6 @@ Cypress.Commands.add('validateExternalLink', { prevSubject: ['optional', 'elemen
 
 installCustomPercyCommand({
   elementOverrides: {
-    '[data-cy=top-nav-cypress-version-current-link]': ($el) => {
```

Comment:
No need to set a dummy value here (vX.X.X). We [mock the value](https://github.com/cypress-io/cypress/blob/ff0d268fb07b85b529edf1290d5a51193f85b338/packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts#L337) that comes back from this and return the value directly from our root `package.json` file, so this will only update when we create the PR to bump the version, which is expected.

Otherwise, the snapshot should remain stable.